### PR TITLE
Schema Integration for [Section Name] = "featured-blogs"

### PR DESCRIPTION
**Pull request by kmalap05**
Issue No: #62 
Schema Integration for [Section Name] = "featured-blogs"

image
image
image

### CODE REVIEW

**Comment by mihir-bombay-studio** [comment]
can you add some liquid html that can display the blog posts inside the section. Basic structure will do

**Comment by kmalap05** [comment]
Changes made.

File: sections/featured-blogs.liquid [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<section
+  class="custom-blog-section"
+  style="padding-top: {{ section.settings.padding_top }}px; padding-bottom: {{ section.settings.padding_bottom }}px;"
```
Comment (mihir-bombay-studio) [inline sections/featured-blogs.liquid:3]: we don't need to add a section element as shopify itself creates a section wrapper for our sections by default, we can aslo add custom classes to that wrapper from the schema(refer documentation). Adding inline styles is not recommended unless necessary.

File: sections/featured-blogs.liquid [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  <div class="custom-blog-section__container">
+    {% for post in blogs[section.settings.blog].articles limit: section.settings.post_limit %}
+      <article class="custom-blog-post" style="background-color: {{ section.settings.color_scheme }};">
+        {% if section.settings.show_image and post.image %}
+          <img
+            src="{{ post.image.src | img_url: 'medium' }}"
+            alt="{{ post.image.alt }}"
+            class="custom-blog-post__image"
+            loading="eager"
+          >
```
Comment (mihir-bombay-studio) [inline sections/featured-blogs.liquid:20]: use image_tag filter instead

File: sections/featured-blogs.liquid [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        </div>
+      </article>
+    {% endfor %}
+  </div>
+
+  {% if section.settings.show_view_all %}
+    <a
+      href="https://theme-skeleton-project.myshopify.com/blogs/{{ blogs[section.settings.blog] }}"
+      class="custom-blog-section__view-all"
+      >View All</a
```
Comment (mihir-bombay-studio) [inline sections/featured-blogs.liquid:39]: Url's and text must not be hardcoded, try taking these values from the schema.

**Review by mihir-bombay-studio — CHANGES_REQUESTED** [review]
@kmalap05 Sorry for the late review. Can you check the comments and fix them? Thanks!

**Comment by kmalap05** [comment]
Please verify, I have made all the changes requested.

**Review by mihir-bombay-studio — APPROVED** [review]

**Pull request merged**